### ISSUE TITLE
jettywrapper 2.0.1 fails on zip

### ISSUE THREAD

**Issue by jcoyne** [projecthydra/jettywrapper#39]
cc @barmintor 
```
rake aborted!
LoadError: cannot load such file -- zip
/home/travis/build/projecthydra/active_fedora/gemfiles/vendor/bundle/ruby/2.2.0/gems/jettywrapper-2.0.1/lib/jettywrapper.rb:13:in `require'
/home/travis/build/projecthydra/active_fedora/gemfiles/vendor/bundle/ruby/2.2.0/gems/jettywrapper-2.0.1/lib/jettywrapper.rb:13:in `<top (required)>'
```

**Comment by jcoyne** [comment]
See https://travis-ci.org/projecthydra/active_fedora/jobs/49997109

**Comment by barmintor** [comment]
Aargh, rubyzip is a development dependency.

**Comment by atz** [comment]
Fixed in `2.0.4`
